app/controllers/facilities_controller.rb

```diff
@@ -37,8 +31,11 @@ def dashboard
 
   # GET /facilities
   def index
-    @facilities = Facility.active.sorted
+    @facilities = Facility.active.alphabetized
+    @recently_used_facilities = MostRecentlyUsedSearcher.new(acting_user).recently_used_facilities.alphabetized
     @active_tab = "home"
+    # recent products get sorted alphabetically in the product_list partial
+    @recent_products = MostRecentlyUsedSearcher.new(acting_user).recently_used_products.includes(:facility)
```

Comment:
Is there any downside to adding the alphabetization here, even if it's still re-sorted in the partial. We don't need to do it now, but I think it would be nice to not do the sort in the `product_list` partial (`facilities/show` is kind of nasty already, but could defiitely be cleaned up), but that can be for another time.